### ISSUE TITLE
feature: auto-populate csv import field dropdowns by default

### ISSUE THREAD

**Issue by whatisgalen** [archesproject/arches#10391]
We could probably leverage ES or a database query to automatically guess which headers/column names match to which fields (node aliases). It might also be nice to save a mapping on the frontend that you could "favorite"

**Comment by chiatt** [comment]
Yeah, it would be nice to make this easier if you have to frequently import data. These are really two different features suggested here, though. I think saving mappings should be handled by a separate issue/pr.
